src/components/Common/Head.js

```diff
@@ -29,8 +29,13 @@ const Head = ({ page }) => {
 
         const description = page.seoMetaDescription || page.seoDescription
 
+        /**
+         * Contentful doesn't give us https: in the image urls so
+         * we have to prefix them here for twitter et al to pick
+         * up the image properly
+         */
         const imageUrl = page.socialLogo
-          ? page.socialLogo
+          ? `https://${page.socialLogo}`
```

Comment:
results in `https:////images.ctfassets.net/22g1lenhck4z/18KTBFOFsaIE6gcMQQMWkk/792ebef4cd9b995618b11aa8d09bd56c/trainline_export__1_.png` (I believe the default started with `//` before)